exla/Makefile

```diff
@@ -44,8 +44,13 @@ $(EXLA_SO): $(EXLA_CACHE_SO)
 		cp -a $(abspath $(XLA_EXTENSION_LIB)) $(EXLA_LIB_DIR) ; \
 		cp -a $(abspath $(EXLA_CACHE_SO)) $(EXLA_SO) ; \
 	else \
-		ln -sf $(abspath $(XLA_EXTENSION_LIB)) $(EXLA_LIB_DIR) ; \
-		ln -sf $(abspath $(EXLA_CACHE_SO)) $(EXLA_SO) ; \
+		if ! command -v realpath &> /dev/null; then \
+			ln -sf $(abspath $(XLA_EXTENSION_LIB)) $(EXLA_LIB_DIR) ; \
+			ln -sf $(abspath $(EXLA_CACHE_SO)) $(EXLA_SO) ; \
+		else \
+			ln -sf $(realpath --relative-to=$(EXLA_LIB_DIR) $(XLA_EXTENSION_LIB)) $(EXLA_LIB_DIR) ; \
+			ln -sf $(realpath --relative-to=$(EXLA_SO) $(EXLA_CACHE_SO)) $(EXLA_SO) ; \
+		fi \
```

Comment:
@thiagopromano can you please check if this version works as expected for you?

Reply:
It doesn't actually, it still generates absolute links, I believe that MAKE ends up using `realpath(3)` as said here: https://www.gnu.org/software/make/manual/html_node/File-Name-Functions.html

Anyway, I've just pushed a change that works on both MacOS and Linux, I've tested it locally on both OSs this time 😄 .

Reply:
I didn't see you last comment on Venture, I'll add a check there.